.changes/draggable-regions.md

```diff
@@ -0,0 +1,5 @@
+---
+"wry": minor
```

Comment:
LGTM! Could you update with `patch` since this is not a breaking change?

Reply:
I thought 'major' is for breaking changes and  `minor` for new features.

Reply:
yeah, but that usually means [main version](https://semver.org/#spec-item-6) is >= 1.0. With main version under 1 in this crate, I wish to update minor version only when there's breaking changes.

Reply:
alright, let me change it